libs/core-ui/src/lib/Cohort/CohortInfo/CohortInfo.styles.ts

```diff
@@ -19,7 +19,7 @@ export const cohortInfoStyles: () => IProcessedStyleSet<ICohortInfoStyles> =
         minWidth: "120px"
       },
       container: {
-        padding: "0 0 0 20px"
+        padding: `65px 0 0 20px`
```

Comment:
Why backquotes?

Reply:
I had different logic to get data but then replaced with this. No specific reason.